src/server/status/index.js

```diff
@@ -18,7 +19,11 @@ export default function (kbnServer, server, config) {
   server.route(wrapAuth({
     method: 'GET',
     path: '/api/status',
-    handler: function (request, reply) {
+    handler: async function (request, reply) {
+      const stats = getStats(
+        server.config().get('kibana.index'),
+        request,
+        server.plugins.elasticsearch.getCluster('admin').callWithRequest);
```

Comment:
The branch has since been updated to accept a callAdminCluster function: https://github.com/elastic/kibana/pull/11632/files#diff-cc9fbe8850c8f590da1323a91d29717dR10

You could partially apply callWithRequest or pass callWithInternalUser directly https://github.com/elastic/kibana/pull/11632/files#diff-1ebbe7846164a2558bba9ae8d00e88f7R17